content/docs/portals.md

```diff
@@ -4,21 +4,22 @@ title: Portals
 permalink: docs/portals.html
 ---
 
-Portals provide a first-class way to render children into a DOM node that exists outside the DOM hierarchy of the parent component.
+Los portales proporcionan un camino de primera clase para mostrar los hijos en un nodo del DOM, que existe afuera de la jerarquía del DOM del componente padre.
```

Comment:
I think we should keep "Portals" instead of "Los portales" as Portals works as a subject here. Same as the page's title.

Also, a more accurate translation would be: "Portals proporciona una opción de primera clase para renderizar hijos en un nodo DOM que existe por fuera de la jerarquía del DOM del Componente padre."

Not quite sure about *first-class way* translation for a better meaning in Spanish.